com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -278,25 +278,9 @@ public static void NetworkShow(List<NetworkObject> networkObjects, ulong clientI
                 }
             }
 
-
-            var context = networkManager.MessageQueueContainer.EnterInternalCommandContext(
-                MessageQueueContainer.MessageType.CreateObjects, NetworkChannel.Internal,
-                new[] { clientId }, NetworkUpdateLoop.UpdateStage);
-
-            if (context != null)
+            foreach (var networkObject in networkObjects)
             {
-                using (var nonNullContext = (InternalCommandContext)context)
-                {
-                    nonNullContext.NetworkWriter.WriteUInt16Packed((ushort)networkObjects.Count);
-
-                    for (int i = 0; i < networkObjects.Count; i++)
-                    {
-                        networkObjects[i].Observers.Add(clientId);
-
-                        networkManager.SpawnManager.WriteSpawnCallForObject(nonNullContext.NetworkWriter, clientId,
-                            networkObjects[i]);
```

Comment:
wouldn't this delete and the other below (`nonNullContext.NetworkWriter.WriteUInt64Packed(networkObjects[i].NetworkObjectId);`) break some stuff here? 👀

Reply:
ohh.. right.. I had missed this and the subtle `s` in `CreateObjects`. I think this highlights the need for a test.
Thanks! Will rework.

Reply:
I take back my "ohh... right...".
This removes a `WriteUInt64Packed` and also removes a matching number of `WriteSpawnCallForObject`. The whole thing is done in a `CreateObjects` context. Removing them all is. fine as it form as consistent block.

It its place, we now call a given number of `NetworkShow`. But each will generate a consistent set of bytes. This then works.

I've updated the branch and checked the new test `NetworkShowHideTest`, which passes.

I've also check on my not yet committed spawn branch. It works there too, with all the traffic going in Snapshot Messages.

Can you please re-check and potentially approve ? (but yeah, the test should have been there in the first place)